### DISCUSSION TITLE
Layout customization in Schema for type "object" (oneOf)

### DISCUSSION THREAD

**Discussion by RamasubramanianA** [ngx-formly/ngx-formly#2755] [Q&A] [answered]
https://angular-mgdft6.stackblitz.io

How can we customize this UI layout, I want all three field in one line , Title is not required to be displayed.
Thanks... ! 

![image](https://user-images.githubusercontent.com/10289214/111639585-62ee1300-8821-11eb-8dd0-257b5808445f.png)

**Comment by aitboudad** [comment, marked as answer]
For all your oneOf schema: update `MultiSchemaTypeComponent` component
For a specific field: create a custom type and  replacee the `multischema` type using widget/or map (https://github.com/ngx-formly/ngx-formly/issues/1982#issuecomment-566048527)

**Reply by RamasubramanianA** [reply]
Thanks @aitboudad 
https://stackblitz.com/edit/oneof-formly?file=src/app/app.component.ts

Adding widget doesn't help, Is it my approach right?

**Reply by aitboudad** [reply]
you've missed `formlyConfig`:

```patch
+ "widget": {
+   "formlyConfig": {
+     "className": "col-4"
+   }
```
